src/TrezorKeyring.ts

```diff
@@ -486,27 +550,25 @@ class TrezorKeyring extends EventEmitter {
     this.hdPath = hdPath;
   }
 
-  /* PRIVATE METHODS */
-
-  _normalize(buf) {
+  #normalize(buf: Buffer) {
     return ethUtil.bufferToHex(buf).toString();
   }
 
-  // eslint-disable-next-line no-shadow
-  _addressFromIndex(pathBase, i) {
+  // eslint-disable-next-line
```

Comment:
Nit: May as well leave the rule name here

```suggestion
  // eslint-disable-next-line @typescript-eslint/no-shadow
```
Or we could fix it by renaming the parameter